scripts/stop-admin.sh

```
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+REPOSITORY=/home/ubuntu/spring-github-action
+cd $REPOSITORY
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)